solidity/dashboard/src/css/banner.less

```diff
@@ -123,6 +123,7 @@
     .banner__title {
       &:extend(.h3);
       margin-bottom: 2rem;
+      font-weight: 600;
```

Comment:
Shouldn't we update only for coverage pool banner on the overview page? It will change styles of all banners.

Reply:
91482d3373715dbe087f6d1974a53f69891171e2